documentation/modules/auxiliary/gather/vmware_vcenter_vmdir_ldap.md

```diff
@@ -0,0 +1,100 @@
+## Vulnerable Application
+
+### Description
+
+This module uses an anonymous-bind LDAP connection to dump data from
+the vmdir service in VMware vCenter Server version 6.7 prior to the
+6.7U3f update.
+
+### Setup
+
+Tested in the wild. No setup notes available at this time.
```

Comment:
Can we add a note to this to say setup is specific to target environment therefore we can't provide setup steps atm?